### PR TITLE
Notes/GitHub and markdown

### PR DESCRIPTION
**Pull request by PacoSchwab**
Added commands and hints regarding git and md, please review :)

### CODE REVIEW

File: shell-and-git/git-and-markdown-notes.md [hunk 1]
```diff
@@ -2,3 +2,33 @@ Shell / Terminal: tab Autovervollständigung, pwd: Ordnerübersicht, ls: Unteror
...
+"open ." Datei im Finder öffnen
+
+git switch -c xyz: neuen Branch erstellen (c = create)
+git switch main: zurück zum main branch
+git switch xyz: wechselt zum entsprechenden Branch
+"code .": Öffnet aktuelles Verzeichnis in visualstudio code
+
+cmd+j in visualstudio code: Terminal öffnen
+
+cmd + k: löschen in Terminal
```
Comment (Kevinwey) [inline shell-and-git/git-and-markdown-notes.md:26]: This one helps me a lot thx. :)

File: shell-and-git/git-and-markdown-notes.md [hunk 2]
```diff
@@ -2,3 +2,33 @@ Shell / Terminal: tab Autovervollständigung, pwd: Ordnerübersicht, ls: Unteror
 
 In .gitignore-Datei die Dateien rein schreiben, die nicht ins Repo sollen, z.B. .DS_Store
 
+weitere Commands:
+
+top (ressourcenfressendste Anwendungen anzeigen), man (Anleitung für bestimmte Befehle, z.B. "man git")
+
+in visual studio code (preview von md dateien anzeigen)
```
Comment (Kevinwey) [inline shell-and-git/git-and-markdown-notes.md:9]: missed that one thx. :)

File: shell-and-git/git-and-markdown-notes.md [hunk 3]
```diff
@@ -2,3 +2,33 @@ Shell / Terminal: tab Autovervollständigung, pwd: Ordnerübersicht, ls: Unteror
 
 In .gitignore-Datei die Dateien rein schreiben, die nicht ins Repo sollen, z.B. .DS_Store
```
Comment (Kevinwey) [inline shell-and-git/git-and-markdown-notes.md:3]: Good point :) next time in english.

File: shell-and-git/git-and-markdown-notes.md [hunk 4]
```diff
@@ -2,3 +2,33 @@ Shell / Terminal: tab Autovervollständigung, pwd: Ordnerübersicht, ls: Unteror
...
 
+weitere Commands:
+
+top (ressourcenfressendste Anwendungen anzeigen), man (Anleitung für bestimmte Befehle, z.B. "man git")
```
Comment (Kevinwey) [inline shell-and-git/git-and-markdown-notes.md:7]: sds

File: shell-and-git/git-and-markdown-notes.md (hunk 3 above)
Comment (PacoSchwab) [reply shell-and-git/git-and-markdown-notes.md:3]: Ok, I will do so.

**Pull request merged**